keras_retinanet/models/resnet.py

```diff
@@ -19,66 +19,69 @@
 import keras_resnet.models
 from ..models import retinanet
 
-WEIGHTS_PATH_NO_TOP_50 = 'https://github.com/fizyr/keras-models/releases/download/v0.0.1/ResNet-50-model.keras.h5'
-WEIGHTS_PATH_NO_TOP_101 = 'https://github.com/fizyr/keras-models/releases/download/v0.0.1/ResNet-101-model.keras.h5'
-WEIGHTS_PATH_NO_TOP_152 = 'https://github.com/fizyr/keras-models/releases/download/v0.0.1/ResNet-152-model.keras.h5'
+resnet_filename = 'ResNet-{}-model.keras.h5'
+resnet_resource = 'https://github.com/fizyr/keras-models/releases/download/v0.0.1/{}'.format(resnet_filename)
 
 custom_objects = retinanet.custom_objects.copy()
 custom_objects.update(keras_resnet.custom_objects)
 
 
-def ResNet50RetinaNet(inputs, num_classes, weights='imagenet', *args, **kwargs):
-    image = inputs
+def resnet_retinanet(num_classes, backbone='50', inputs=None, weights='imagenet', **kwargs):
```

Comment:
`backbone` as string seems a bit weird. Why not just use a the numbers 50, 101 and 152?

Reply:
It used to be `'resnet50'` but I changed that later on to `'50'`. But you're right, I'll change it.